src/index.html

```diff
@@ -1,14 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <main class="page">
+      <section class="clock">
+        <div class="clock__minute"></div>
+        <div class="clock__second"></div>
+        <div class="clock__center"></div>
```

Comment:
I think it'll be better with space between. 
```suggestion
        <div class="clock__minute"></div>
        
        <div class="clock__second"></div>
        
        <div class="clock__center"></div>
```

Reply:
fix.